### ISSUE TITLE
Revise description accounts

### ISSUE THREAD

**Issue by MichaelWuestSGKB** [OpenWealth/OpenWealth#129]
https://github.com/OpenWealth/OpenWealth/blob/de9e2aeecf2829665da26fff15e66c8d2ab8af1a/CustodyServices/Docs/EntityModel.md?plain=1#L47-L50

For more clarity regarding the usage of an IBAN, I would rephrase it as follows: 

An account is an entity that describes a basket/container of financial instruments. Each account must be associated to a customer (typically the account holder and beneficial owner) and is usually enclosed in a portfolio. There mainly two types of accounts:

- Cash Account
- Safekeeping Account

An account must by identified by a unique reference, the account number. **Most cash accounts (and some safekeeping accounts?) should also have an IBAN - International Bank Account Number - associated to it**.

The cash account is as commonly understood a container for holdings of a currency […]

**Comment by andiahlm** [comment]
@MichaelWuestSGKB Danke für die Präzisierung, habe das so im Doku-Branch umgesetzt:

https://github.com/OpenWealth/OpenWealth/blob/improve-custody-services-documentation/CustodyServices/Docs/EntityModel.md#account

**Comment by MichaelWuestSGKB** [comment]
@andiahlm Thank you. Is the intention that I close the issue?

**Comment by andiahlm** [comment]
> @andiahlm Thank you. Is the intention that I close the issue?

It will automatically be closed with the reference in the pull-request...
